shadowsocks/lan.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Jigsaw Operations LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shadowsocks
+
+import "net"
+
+var lanSubnets []*net.IPNet
+
+func init() {
+	for _, cidr := range []string{
```

Comment:
Let's reference the RFCs: RFC 1918 for ipv4 and RFC 4193 for ipv6.

Reply:
Done in cd299cd.